app/Exceptions/ProcessFailed.php

```diff
@@ -2,9 +2,12 @@
 
 namespace App\Exceptions;
 
+use Symfony\Component\Process\Process;
```

Comment:
Do we still need/want those changes? If we do, it would be helpful to add type hints to all those methods.